README.md

```diff
@@ -1,14 +1,3 @@
-# DHIS2 end to end testing
-
-Repository for DHIS2 E2E testing.
-
-## Technology stack
-
-* [Webdriver.io](http://webdriver.io/): testing utility working with Selenium or other standalone WebDriver drivers
-* [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/): Webdriver driver for Chrome browser
-* [Cucumber](https://cucumber.io/): BDD test automation framework
-* [Chai](https://chaijs.com/): BDD/TDD assertion library
```

Comment:
Maybe it's worth adding the new technology stack in here. i.e. cypress, allure, report portal, etc?

Reply:
Will do